### PR TITLE
changes to the index page

### PR DESCRIPTION
**Pull request by kennedylinzie**
cosmetic changes to the index page.

### CODE REVIEW

**Comment by kennedylinzie** [comment]
change

File: static/index.html [hunk 1]
```diff
@@ -6,55 +6,122 @@
...
+      <div class="text-center py-5">
+        <img src="../img/squre.jpg" width="200" class="rounded mx-auto d-block" alt="...">
+        <h2 class="display-5 fw-bold">Generate a Custom Calendar</h2>
+        <p class="display-6">Choose the presets for your calender...</p>
+      </div>
+   
+      <div class="row justify-content-center my-">
+        <div class="col-lg-6">
+          
+          <form>
```
Comment (zikani03) [inline static/index.html]: Change the form to use a post to the "/calendar" endpoint

`<form method="post" action="/calendar" enctype="multipart/form-data">`

**Review by zikani03 — CHANGES_REQUESTED** [review]
Looks good. Just fix the form method/action

**Comment by kennedylinzie** [comment]
changes made ,wonder how I missed that haha

**Review by zikani03 — APPROVED** [review]

**Pull request merged**